geo-types/src/line_string.rs

```diff
@@ -149,6 +149,21 @@ impl<T: CoordinateType> LineString<T> {
             }
         }
     }
+
+    /// Return the number of coordinates in the `LineString`.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// use geo_types::LineString;
+    ///
+    /// let mut coords = vec![(0., 0.), (5., 0.), (7., 9.)];
+    /// let line_string: LineString<f32> = coords.into_iter().collect();
+    /// assert_eq!(3, line_string.nb_coords());
+    /// ```
+    pub fn nb_coords(&self) -> usize {
```

Comment:
thoughts on calling this `num_coords`? i personally haven't seen the `nb_` naming convention in the ecosystem before